R/sgdnet.R

```diff
@@ -9,19 +25,21 @@
 #' @param maxit maximum number of effective passes (epochs)
 #' @param standardize whether to standardize `x` or not -- ignored when
 #'   `intercept == FALSE`.
-#' @param thresh tolerance level for termination of the algorithm
-#' @param return_loss whether to compute and return the loss at each outer
-#'   iteration, only added here for debugging purposes.
+#' @param thresh tolerance level for termination of the algorithm. The
+#'   algorithm terminates when
+#'   \deqn{
+#'     \frac{\mathrm{change~in~weights}}{\mathrm{weights}} < \mathrm{thresh}.
```

Comment:
I'd vote to call these "coefficients" since weights is already used to refer to i) observation weights; and ii) feature weights. 

It might be clearer to write this as something like

\frac{\|\beta^{(t)} - \beta^{(t-1)}\|_{\infty}}{\|\beta^{(t)}\|_{\infty}} < \mathrm{thresh}

Reply:
Hm, in that case I think we should probably switch from using weights for feature weights altogether and simply stick to coefficients or betas. It would only be confusing to use beta or coefficient here but then use weights in the internal API.

Reply:
I'd agree, but maybe wait until you're done porting the basics from sklearn to make the initial port easy? Once that's done, a big rename should be pretty straightforward.